tool/build_e2e.dart

```diff
@@ -41,20 +40,23 @@ void main(List<String> args) async {
     [
       'pkg/dds/tool/devtools_server/serve_local.dart',
       '--devtools-build=$devToolsBuildLocation',
+      // Pass any args that were provided to our script along. This allows IDEs
+      // to pass `--machine` (etc.) so that this script can behave the same as
+      // the "dart devtools" command for testing local DevTools/server changes.
+      ...args,
     ],
     workingDirectory: localDartSdkLocation,
   );
-  _printToConsole(serveProcess);
+  _forwardOutputStreams(serveProcess);
+  _forwardInputStream(serveProcess);
   await serveProcess.exitCode;
 }
 
-void _printToConsole(Process process) {
-  _transformToLines(process.stdout).listen(print);
-  _transformToLines(process.stderr).listen(print);
+void _forwardOutputStreams(Process process) {
+  process.stdout.listen(stdout.add);
+  process.stderr.listen(stdout.add);
```

Comment:
are stdout and stderr still printed to console like they used to be?

Reply:
Yep, this is essentially the same as before - anything that comes from `process.stdout` or `process.stderr` gets piped to `stdout`. This way sends the bytes as they come rather than waiting for a newline and then using `print` though (which isn't significant, but it felt slightly more consistent with the input one below).